src/components/Hero/Hero.jsx

```diff
@@ -28,8 +28,18 @@ export default function Hero() {
 		<div className="flex justify-center items-center h-screen w-full bg-square-svg bg-center bg-cover bg-no-repeat bg-fixed relative md:bg-scroll">
 			<div className="flex flex-col justify-center items-center gap-12 translate-y-8 z-10 md:gap-6 max-w-2xl">
 				<div id="title" className="flex items-center justify-center" data-aos="fade-up">
-					<img id="hth-logo-outline" {...HtHoutline} alt="Hack the Hill Logo" className="absolute w-3/4" />
-					<img id="hth-logo-solid" {...HtHSolid} alt="Hack the Hill Logo" className="relative w-3/4" />
+					<img
+						id="hth-logo-outline"
+						{...HtHoutline}
+						alt="Capital Technology Network Logo"
+						className="absolute w-3/4"
+					/>
+					<img
+						id="hth-logo-solid"
+						{...HtHSolid}
+						alt="Capital Technology Network Logo"
+						className="relative w-3/4"
+					/>
```

Comment:
This doesn't actually change what is displayed 
So the website still looks like Hack the Hill not the Capital Technology Network...
Maybe we need to replace this section with text rather than an image, if we don't have an updated logo